google_cloud_compute/plugin.spec.yaml

```diff
@@ -1373,23 +1354,27 @@ actions:
         title: Filter
         description: Sets a filter expression for filtering listed resources
         required: false
+        example: name = my-firewall-1
       maxResults:
         type: integer
         title: Max Results
         description: The maximum number of results per page that should be returned.
           Acceptable values are 0 to 500. Default 500
         required: false
+        example: 0
       orderBy:
         type: string
         title: OrderBy
         description: Sorts list results by a certain order
         required: false
+        example: creationTimestamp desc
       pageToken:
         type: string
         title: Page Token
         description: Set pageToken to the nextPageToken returned by a previous list
           request to get the next page of results
         required: false
+        example: CkYIpsyE6vCM7QI6OwoCGAEKAiAACgIYAgoHINDkjPrMGgoCGAIKFCoSZGVmYXVsdC1hbGxvdy1pY21wCgoggvCxhNin4rIJ
```

Comment:
Can we remove this input and handle paging automatically for the user in the code?

Reply:
Hey @jschipp-r7 , thank you for the review. We've change the title and description, apologies for missing that in the previous commit. About the "pagination" comments, could we please follow-up on them in the new ticket? It looks like a few hours of work (with testing etc). We do agree that it should be handled internally without user knowledge, but we don't want to block this PR. In addition, based on GCP documentation, the default value for "maxResults" field is 500, so it seems unlikely that many users will run into this problem.

Can we follow-up this on new ticket?

Reply:
I created a ticket for this - let's do it here so we can release once without another major version bump